### DISCUSSION TITLE
Can the Tycho build extension include the folders in the root folder automatically?

### DISCUSSION THREAD

**Discussion by vogella** [eclipse-tycho/tycho#2435] [General]
Is it possible to include the folders (containing plug-ins, features, etc) of the root folder automatically in the build via the Tycho build extension? AFAICS it works for sub-folders, e.g. if I have a bundles folder it child modules can be automatically included.

**Comment by laeubi** [comment]
If you have a `bundles` folder and mention this folder in the `<modules>` section all of them are included.
This does not work for the "root" as mane requires a module to be present for Tycho to do something (e.g. generate an aggregate).
